tests/components/test_population.py

```diff
@@ -0,0 +1,188 @@
+            pd.Timestamp("1983-01-31 00:00:00"),
+            pd.Timestamp("2025-04-30 00:00:00"),
+            pd.Timestamp("1990-01-11 00:00:00"),
+        ],
+        "age": [68, 44, 7, 9, 38],
+        "guardian_1": [-1, -1, 100, 100, -1],
+        "relation_to_household_head": [
+            "Parent-in-law",
+            "Same-sex spouse",
+            "Biological child",
+            "Biological child",
+            "Sibling",
+        ],
+        "housing_type": ["Standard"] * 5,
+    }
+)
+
+fake_household_3 = pd.DataFrame(
+    {
+        "household_id": [3] * 5,
+        "date_of_birth": [
+            pd.Timestamp("1965-11-08 00:00:00"),
+            pd.Timestamp("1992-05-18 00:00:00"),
+            pd.Timestamp("1967-02-05 00:00:00"),
+            pd.Timestamp("1995-04-28 00:00:00"),
+            pd.Timestamp("2028-08-13 00:00:00"),
+        ],
+        "age": [63, 27, 60, 23, 2],
+        "guardian_1": [-1, -1, -1, -1, 35],
+        "relation_to_household_head": [
+            "Parent",
+            "Sibling",
+            "Parent",
+            "Opp-sex spouse",
+            "Biological child",
+        ],
+        "housing_type": ["Standard"] * 5,
+    }
+)
+
+fake_household_4 = pd.DataFrame(
+    {
+        "household_id": [4] * 5,
+        "date_of_birth": [
+            pd.Timestamp("1995-04-22 00:00:00"),
+            pd.Timestamp("1996-07-02 00:00:00"),
+            pd.Timestamp("1996-04-22 00:00:00"),
+            pd.Timestamp("1996-09-16 00:00:00"),
+            pd.Timestamp("1997-10-30 00:00:00"),
+        ],
+        "age": [23, 22, 22, 21, 20],
+        "guardian_2": [-1, -1, -1, -1, -1],
+        "relation_to_household_head": [
+            "Roommate",
+            "Roommate",
+            "Roommate",
+            "Roommate",
+            "Sibling",
+        ],
+        "housing_type": ["Standard"] * 5,
+    }
+)
+
+fake_household_5 = pd.DataFrame(
+    {
+        "household_id": [5] * 5,
+        "date_of_birth": [
+            pd.Timestamp("1988-12-22 00:00:00"),
+            pd.Timestamp("2025-04-11 00:00:00"),
+            pd.Timestamp("1990-02-14 00:00:00"),
+            pd.Timestamp("1992-08-15 00:00:00"),
+            pd.Timestamp("1990-04-01 00:00:00"),
+        ],
+        "age": [38, 5, 36, 34, 36],
+        "guardian_2": [-1, 20, -1, -1, -1],
+        "relation_to_household_head": [
+            "Opp-sex spouse",
+            "Biological child",
+            "Roommate",
+            "Other relative",
+            "Other nonrelative",
+        ],
+        "housing_type": ["Standard"] * 5,
+    }
+)
+
+
+@pytest.fixture(scope="session")
+def fake_population() -> pd.DataFrame:
+    return pd.concat(
+        [
+            fake_household_1,
+            fake_household_2,
+            fake_household_3,
+            fake_household_4,
+            fake_household_5,
+        ]
+    ).reset_index()
+
+
+def test_update_to_reference_person_and_relationships(fake_population):
+
+    expected_relationships = pd.Series(
```

Comment:
My idea with splitting the households was that you could then split the expected relationships, which would allow someone to more easily follow _why_ these are the expected relationships.

Reply:
Ok I will split this up and concat it.